src/styles/components/navbar.scss

```diff
@@ -8,7 +8,8 @@
     flex-direction: row;
     align-items: center;
     height: $height-promotion;
-    background-color: $B3;
+    background-image: url("../../../images/promotion-banner-background-image.svg");
```

Comment:
why change background color to image? I think it's easier to use background-color.

Reply:
the background is a gradient color, and the gradient color is not linear gradients, so use the image provided by UI designer